Packs/FlashpointFeed/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,25 @@
+
+#### Indicator Fields
+
+##### New: FlashpointFeed HTML Malware Description
+
+- New: Added the HTML based indicator field for the malware description.
+
+
+#### Layouts
+
+##### Flashpoint Indicator
+
+- Updated the layout to include the new HTML based indicator field for Malware Description.
```

Comment:
If markdown is used you can use the ootb indicators which will be better for your XSOAR customers.